tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./node_modules/@salesforce/dev-config/tsconfig",
+  "extends": "./node_modules/@salesforce/dev-config/tsconfig.json",
```

Comment:
**question:** why should we add the extension here?

Is it solving something ?

Reply:
tsconfig.json change shouldn't be there. Please ignore it.

checking the solution with `export = sgd`...